src/components/options/providers/groups.svelte

```diff
@@ -1,41 +1,61 @@
 <script>
+import _ from "lodash";
 import Notiflix from "notiflix";
 import { createEventDispatcher } from "svelte";
 
 import LocalStore from "../../../js/shared/local_store";
 import { StoreKey } from "../../../js/shared/constants";
 
-// Props.
-export let initialSettings;
+const dispatch = createEventDispatcher();
+
+// Auxiliary variable to store initial settings.
+let initialSettings;
```

Comment:
This variable should be bellow `// States` since changing it will force the UI to be redraw.

Also, you should rename it to `initialGroups` and store in them only the `providersGroups` fields (since you aren't using any other field).